lib/car.rb

```diff
@@ -0,0 +1,36 @@
+class Car
+
+  def initialize(make, monthly_payment, loan_length)
+    @make = make
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+  end
+
+  def make
+    @make = "Ford"
+  end
+
+  def model
+    @model = "Mustang"
+  end
+
+  def monthly_payment
+    @monthly_payment = 1500
+  end
+
+  def loan_length
+    @loan_length = 36
+  end
```

Comment:
It looks like in a lot of your methods you are using the assignment operator `=`. What this will do is **reassign** the value of `@loan_length`. We don't want to do that because that will permanently change that variable. Instead we just want to return the loan length, so this method could be written as

```
def loan_length
  @loan_length
end
```

Reply:
Also, once this idea clicks, review previous lessons to see how we can do this same thing with an `attr_reader`